tests/mfx_dispatch/linux/CMakeLists.txt

```diff
@@ -22,7 +22,12 @@
 # library implementation inside. A key thing for the trick to work is
 # avoiding non-default symbol versioning, i.e. -Wl,--default-symver which
 # is propagated by mediasdk test system should be avoided. 
-add_executable(mfx_dispatch_tests mfx_dispatch_tests.cpp)
+include_directories(include)
```

Comment:
maybe we will keep all files in a single folder? why we need 'include'?

Reply:
Usually source and include files directories are kept separte.

Reply:
Include are sometimes kept separate if that's externally exposed uAPI. In other cases, for internal things they are usually kept together:
- https://github.com/FFmpeg/FFmpeg/tree/master/libavcodec
- https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/tree/drivers/gpu/drm/i915

In this case, for the test, I think better to keep everything together. This will be simpler.